.github/ISSUE_TEMPLATE/4-REGULAR_RELEASE_CHECKLIST.md

```diff
@@ -116,50 +152,54 @@ assignees: 'schlessera'
     md5 hash verified: 179fc8dacbfe3ebc2d00ba57a333c982
     New version works. Proceeding to replace.
     Success: Updated WP-CLI to 1.4.0.
-    $ wp @daniel option get home
```

Comment:
This was meant as a smoke test...

Reply:
A smoke test that only works on your machine :-)

What‘s a good alternative?

Reply:
How about:
```
$ wp cli version
WP-CLI 2.8.1
$wp eval 'echo \WP_CLI\Utils\http_request( "GET", "https://api.wordpress.org/core/version-check/1.6/" )->body;' --skip-wordpress
<PHP serialized string with version numbers>
```
That would at the very least test whether the update worked and whether regular execution and the integration with Requests works.
Note: The above example that Daniel provided mostly just tested the SSH subsystem. The entire option retrieval logic is executed on whatever version is installed on his server.

Reply:
Added in c532c80a